src/components/SendRawTransaction.js

```diff
@@ -16,33 +16,45 @@ class SendRawTransaction extends Component {
   constructor(props) {
     super(props);
     this.state = {
+      hex: '',
       data: 'null'
     };
+    this.handleSubmit = this.handleSubmit.bind(this);
   }
 
   handleInputChange(e) {
     let value = e.target.value;
     this.setState({
-      txid: value
+      hex: value
     });
   }
 
   handleSubmit(e) {
-    // BITBOX.RawTransactions.SendRawTransaction(this.state.txid).then((result) => {
-    //   this.setState({
-    //     data: result
-    //   })
-    // }, (err) => { console.log(err); });
+    BITBOX.RawTransactions.sendRawTransaction(this.state.hex).then((result) => {
+      this.setState({
+        data: result
```

Comment:
You might get an error here. If so you'll need to `stringify` the result.

React will complain when you try and get it to render JSON. So you have to:

`data: JSON.stringify(result)`